test/e2e/adminapi_redeployvm.go

```diff
@@ -41,6 +41,12 @@ var _ = Describe("[Admin API] VM redeploy action", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(vms).NotTo(BeEmpty())
 		vm := vms[0]
+		for _, possibleVM := range vms {
+			if strings.Contains(*possibleVM.Name, "-master-") || strings.Contains(*possibleVM.Name, "-worker-") {
+				vm = possibleVM
+				break
+			}
+		}
```

Comment:
I think the normal cluster has only masters and workers.
In what case do we need this check?

Also I thought we should be able to redeploy any type of VM in the cluster.
Is it not expected to redeploy other than master or worker?

Reply:
well, during my test, I put a jumphost in the master subnet, which is probably not very common and it was then picked by the selection and redeployed, error msg:

```
  [FAILED] Timed out after 600.000s.
  The function passed to Eventually failed at /home/jeff/go/src/redhat/ARO-RP/test/e2e/helpers.go:45 with:
  Unexpected error:
      <*errors.StatusError | 0xc000b29cc0>: 
      pods "jeff-jumphost-1-uptime-1" not found
      {
          ErrStatus: {
              TypeMeta: {Kind: "", APIVersion: ""},
              ListMeta: {
                  SelfLink: "",
                  ResourceVersion: "",
                  Continue: "",
                  RemainingItemCount: nil,
              },
              Status: "Failure",
              Message: "pods \"jeff-jumphost-1-uptime-1\" not found",
              Reason: "NotFound",
              Details: {
                  Name: "jeff-jumphost-1-uptime-1",
                  Group: "",
                  Kind: "pods",
                  UID: "",
                  Causes: nil,
                  RetryAfterSeconds: 0,
              },
              Code: 404,
          },
      }
  occurred

```